operations/flags.go

```diff
@@ -21,9 +21,10 @@ const (
 	numWorkersFlag = "workers"
 	bucketNameFlag = "bucket"
 
-	dbURIFlag       = "dbUri"
-	dbNameFlag      = "dbName"
-	dbCredsFileFlag = "dbCreds"
+	dbURIFlag             = "dbUri"
+	dbNameFlag            = "dbName"
+	dbCredsFileFlag       = "dbCreds"
+	dbCredsCollectionFlag = "dbCredsCollection"
```

Comment:
To me it makes sense to call it the `dbConfigurationCollection` or something since it's not just creds in it.

Reply:
+1